sapphire/simulations/groundparticles.py

```diff
@@ -245,7 +241,7 @@ def simulate_detector_mips_for_particles(self, particles, detector,
             # and the time it took for the first photon to arrive at the PMT.
             file = np.genfromtxt("RUN_1/outpSD.csv", delimiter=",")
             try:
-                numberofphotons = len(file[:,1])-1 # first element is header
+                numberofphotons = len(file[1:,1]) # first element is header
```

Comment:
Instead of slicing use the `skip_header` keyword for `genfromtxt`. Just set it to 1 ("The number of lines to skip at the beginning of the file.")